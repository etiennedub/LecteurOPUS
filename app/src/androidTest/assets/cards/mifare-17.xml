<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Mifare><Pages><List index="0">04C099D5</List><List index="1">2AE34380</List><List index="2">0A48E000</List><List index="3">0000000F</List><List index="4">00000000</List><List index="5">30DF0523</List><List index="6">00000000</List><List index="7">88A3CA7D</List><List index="8">01C3056E</List><List index="9">56E01004</List><List index="10">48000000</List><List index="11">2028326A</List><List index="12">01B6C3BA</List><List index="13">3DC01604</List><List index="14">08008000</List><List index="15">2228481D</List></Pages></Mifare>