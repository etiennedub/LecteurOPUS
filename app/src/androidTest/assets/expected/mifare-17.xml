<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Card><Type>OCCASIONNELLE</Type><ID>0</ID><Trip><List index="0" Date="04/10/2016" Time="11:35" Transfert="false" OperatorId="20" BusId="1" OperatorName="RTC" BusName="LEBUS 1" /><List index="1" Date="22/03/2016" Time="08:14" Transfert="true" OperatorId="20" BusId="1" OperatorName="RTC" BusName="LEBUS 1" /></Trip><Contract><List index="0" Subscription="false" Valid="true" NbTicket="0" OperatorId="20" OperatorName="RTC" /></Contract></Card>